C/simplicity_alloc.h

```diff
@@ -0,0 +1,12 @@
+#ifndef SIMPLICITY_SIMPLICITY_ALLOC_H
+#define SIMPLICITY_SIMPLICITY_ALLOC_H
+
+#include <stdlib.h>
+
+/* Allocate with malloc by default. */
+#define simplicity_malloc(size) malloc(size)
```

Comment:
This can be simply

    #define simplicity_malloc malloc

and similarly for free